cocos/3d/lod/lodgroup-component.ts

```diff
@@ -242,6 +242,11 @@ export class LODGroup extends Component {
 
     private _eventRegistered = false;
 
+    /**
+     * @engineInternal
+     */
+    protected _forceUsedLevels: number[] = [];
```

Comment:
Why does protected property need to mark as `@engineInternal`?

Reply:
Developers can use it by inheritance.

Reply:
Why not use private?